lib/hanyutils.ex

```diff
@@ -120,4 +145,79 @@ defmodule Hanyutils do
     |> Pinyin.read!(:words)
     |> Pinyin.numbered()
   end
+
+  @doc """
+  Convert a string with marked Pinyin to numbered Pinyin.
+
+  Parses the input using `Pinyin.read!/1` (in `:words` mode), and converts the result with
```

Comment:
Perhaps you could add the `mode` as an argument and use `:words` as its default value? Not sure if this makes sense for zhuyin.

Reply:
Should we add that for pinyin, too then?